src/systemctl/systemctl-compat-shutdown.c

```diff
@@ -36,7 +36,7 @@ static int shutdown_help(void) {
                "  -c             Cancel a pending shutdown\n"
                "     --show      Show pending shutdown\n"
                "\n%sThis is a compatibility interface, please use the more powerful 'systemctl reboot',\n"
-               "'systemctl poweroff', 'systemctl reboot' commands instead.%s\n"
+               "'systemctl poweroff' commands instead.%s\n"
```

Comment:
The extra command should stay, but it should say `systemctl halt` instead, since that's missing from the help message.